src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationViewTaggerProvider.cs

```diff
@@ -29,8 +29,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
     /// </summary>
     [Export(typeof(IViewTaggerProvider))]
     [TagType(typeof(IClassificationTag))]
-    [ContentType(ContentTypeNames.CSharpContentType)]
-    [ContentType(ContentTypeNames.VisualBasicContentType)]
+    [ContentType(ContentTypeNames.RoslynContentType)]
```

Comment:
I noticed in @mgoertz-msft's PR #12948 there was a bunch of places Xaml was added to a set of `[ContentType()]` blocks.  Maybe we could apply this approach more widely?